plugins/dra-ocp-validator/tools/cleanup-dra.sh

```
@@ -0,0 +1,254 @@
+#=========================================
+# Step 1: Clean up test namespaces
+#=========================================
+echo "=== Step 1: Clean up test namespaces ==="
+
+TEST_NAMESPACES=$(oc get namespaces -o name 2>/dev/null | grep 'namespace/dra-.*-test' | sed 's|namespace/||' || true)
+
+if [ -z "${TEST_NAMESPACES}" ]; then
+    echo "No test namespaces found (pattern: dra-*-test)"
+else
+    echo "Found test namespaces:"
+    echo "${TEST_NAMESPACES}" | sed 's/^/  - /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)